### PR TITLE
Edit wording for clarity

### PR DESCRIPTION
**Pull request by mjgutermuth**
The wording seemed a little awkward here, so this is my attempt to clarify it. I also removed some extra words.

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/cpu-management-policies.md [hunk 1]
```diff
@@ -71,15 +71,15 @@ gate for each individual option.
 
 ### Changing the CPU Manager Policy
 
-Since the CPU manger policy can only be applied when kubelet spawns new pods, simply changing from
-"none" to "static" won't apply to existing pods. So in order to properly change the CPU manager
+Since the CPU manger policy can only be applied when kubelet spawns new pods, changing from
```
Comment (Kartik494) [inline content/en/docs/tasks/administer-cluster/cpu-management-policies.md:74]: i think it's a typo here shouldn't this be `Since the CPU manager`?

Comment (mjgutermuth) [reply content/en/docs/tasks/administer-cluster/cpu-management-policies.md:74]: Yes, that should be `manager`

Comment (Kartik494) [reply content/en/docs/tasks/administer-cluster/cpu-management-policies.md:74]: I don't see your commits for this change. Could you add the commit here .
Thanks!

Comment (mjgutermuth) [reply content/en/docs/tasks/administer-cluster/cpu-management-policies.md]: Added. Thanks!

**Comment by jihoon-seo** [comment]
@mjgutermuth GitHub says that there is a merge conflict. It would be great if a rebase is performed.
/label tide/merge-method-squash

**Comment by tengqm** [comment]
Merge conflict for over 2 months.
/close

Feel free to reopen and resume this work.

**Pull request closed without merging**